Pipfile

```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 rx = "*"
 pandas = "*"
+pytest = "*"
```

Comment:
I would suggest moving pytest to dev-packages by:

```console
pipenv install --dev pytest
```

Reply:
thanks